Source/WebCore/page/Quirks.cpp

```diff
@@ -365,6 +365,15 @@ bool Quirks::shouldAvoidUsingIOS13ForGmail() const
 #endif
 }
 
+// mail.google.com rdar://128360054
```

Comment:
Nit - maybe instead of listing the bug number for the quirk here, we could turn it into a FIXME; something like:

```
// FIXME (<rdar or bugzilla bug>): Remove this quirk once Gmail adopts the `writingsuggestions` attribute.
```

Reply:
This is done throughout the file to easily indicate the (possibly multiple sites) reasons behind a quirk.

But a FIXME underneath that is good!

Reply:
Added the FIXME underneath in https://github.com/WebKit/WebKit/commit/0d0f6d87804037f405b74be05b723516f4c1268c